scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,702 @@
+    "\n",
+    "The gist of it is: use a model you already have and use ML Engine’s online prediction service to support scalable prediction requests against your trained model.\n",
+    "\n",
+    "This notebook use the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to create a simple model, train the model, upload the model to ML Engine, and lastly use the model to make predictions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to bring your model to ML Engine\n",
+    "Getting your model ready for predictions can be done in 5 steps:\n",
+    "1. Save your model\n",
+    "1. Upload the saved model to [Google Cloud Storage](https://cloud.google.com/storage)\n",
+    "1. Create a model resource on ML Engine\n",
+    "1. Create a model version (linking your Scikit Learn model)\n",
+    "1. Make an online prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prerequisites\n",
+    "Before you jump in, let’s cover some of the different tools you’ll be using to get online prediction up and running on ML Engine. \n",
+    "\n",
+    "[Google Cloud Platform](https://cloud.google.com/) lets you build and host applications and websites, store data, and analyze data on Google's scalable infrastructure.\n",
+    "\n",
+    "[ML Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.\n",
+    "\n",
+    "[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is unified object storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.\n",
+    "\n",
+    "[The Cloud SDK](https://cloud.google.com/sdk/) is a set of tools for Cloud Platform. It contains gcloud, gsutil, and bq, which you can use to access Google Compute Engine, Google Cloud Storage, Google BigQuery, and other products and services from the command-line.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 0: Setup\n",
+    "* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)\n",
+    "* [Create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/quickstart-console)\n",
+    "* [Enable Cloud Machine Learning Engine and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component&_ga=2.217405014.1312742076.1516128282-1417583630.1516128282)\n",
+    "\n",
+    "These variables will be needed for the following steps.\n",
+    "\n",
+    "** Replace: **\n",
+    "* `PROJECT_ID <true-ability-192918>` - with your project's id. Use the PROJECT_ID that matches your Google Cloud Platform project.\n",
```

Comment:
true-* is your project? In which case replace this with something like $YOUR_GCP_PROJECT_ID

Reply:
Fixed